test/functional/p2p_ibd_txrelay.py

```diff
@@ -28,6 +42,30 @@ def run_test(self):
             assert node.getblockchaininfo()['initialblockdownload']
             self.wait_until(lambda: all(peer['minfeefilter'] == MAX_FEE_FILTER for peer in node.getpeerinfo()))
 
+        self.log.info("Check that nodes don't send getdatas for transactions while still in IBD")
+        peer_inver = self.nodes[0].add_p2p_connection(P2PDataStore())
+        txid = 0xdeadbeef
+        peer_inver.send_and_ping(msg_inv([CInv(t=MSG_WTX, h=txid)]))
+        # assert peer_inver doesn't receive a getdata for the txid
+        with p2p_lock:
+            assert txid not in peer_inver.getdata_requests
```

Comment:
I don't think this is testing anything. If a node receives an INV for a transaction from an inbound connection, it won't request it for at least two seconds (see `AddTxAnnouncement()`), so this assert would pass even if the node wasn't in IBD.

Reply:
Oopsie! Added a sleep + sync with ping

Reply:
Thanks. Looks good.